.vscode/cspell.json

```diff
@@ -81,7 +81,6 @@
     "sdk/servicefabric/azure-servicefabric/**",
     "sdk/search/azure-search-documents/**",
     "sdk/storage/azure-storage-blob-changefeed/**",
-    "sdk/servicebus/azure-servicebus/**",
```

Comment:
It looks like removing this results in the [cspell check in our CI failing](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1850286&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0&l=33) on various queue names. 

To work around this, we can just ignore any spelling issues in the mgmt_tests and we should be good to go. You can replace this line with:
`"sdk/servicebus/azure-servicebus/tests/mgmt_tests/**",`